enterprise/frontend/src/metabase-enterprise/api/upload-management.ts

```diff
@@ -0,0 +1,30 @@
+import { Api } from "metabase/api";
+import { provideTableListTags, invalidateTags, tag } from "metabase/api/tags";
+import type {
+  DeleteUploadTableRequest,
+  UploadManagementResponse,
+} from "metabase-types/api";
+
+export const uploadManagementApi = Api.injectEndpoints({
+  endpoints: builder => ({
+    listUploadTables: builder.query<UploadManagementResponse, void>({
+      query: () => ({
+        method: "GET",
+        url: "/api/ee/upload-management/tables",
+      }),
+      providesTags: (tables = []) => provideTableListTags(tables),
```

Comment:
I'm not quite sure this is correct 🤔 this only loads a subset of tables - could it prevent /api/tables list from fetching?

Reply:
i think this is actually correct. the only issue with this is invalidating table lists tags due to a non-csv related table being mutated will cause a refetch here when you maybe don't mean to. i think this is just fine.